src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -83,10 +84,16 @@ protected boolean checkWireTransferTabState(DisbursementVoucherDocument dvDocume
         PaymentSourceWireTransfer dvWireTransfer = dvDocument.getWireTransfer();
 
         // if payment method is CHECK and wire tab contains data, ask user to clear tab
-        if ((StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_CHECK, dvDocument.getDisbVchrPaymentMethodCode()) || StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_DRAFT, dvDocument.getDisbVchrPaymentMethodCode())) && hasWireTransferValues(dvWireTransfer)) {
-            String questionText = SpringContext.getBean(ConfigurationService.class).getPropertyValueAsString(CUKFSKeyConstants.QUESTION_CLEAR_UNNEEDED_WIRE_TAB);
-
-            boolean clearTab = super.askOrAnalyzeYesNoQuestion(KFSConstants.DisbursementVoucherDocumentConstants.CLEAR_WIRE_TRANSFER_TAB_QUESTION_ID, questionText);
+        if ((StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_CHECK,
+                dvDocument.getDisbVchrPaymentMethodCode()) || StringUtils.equals(
+                KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_DRAFT,
+                dvDocument.getDisbVchrPaymentMethodCode())) && hasWireTransferValues(dvWireTransfer)) {
+            String questionText = SpringContext.getBean(ConfigurationService.class).getPropertyValueAsString(
+                    KFSKeyConstants.QUESTION_CLEAR_UNNEEDED_TAB);
```

Comment:
Our old cu-kfs code was using QUESTION_CLEAR_UNNEEDED_WIRE_TAB instead of QUESTION_CLEAR_UNNEEDED_TAB. Can you please confirm if our code still needs to use the former?

Reply:
This change was done as part of KFSPTS-4366, will revert to locally customized code